fail2ban: not building for 25.12.0-rc4

### Package Name

fail2ban

### Maintainer

undefined

### OpenWrt Version

25/12/0-rc4

### OpenWrt Target/Subtarget

ipq40xx/generic

### Steps to Reproduce

1. Build openwrt with fail2ban selected


### Actual Behaviour

2. Watch it fail with
```
make[2]: Entering directory '/home/dwardo/GADGETS/ROUTEURS/openwrt/feeds/packages/net/fail2ban'
rm -f /home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0/.built
touch /home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0/.built_check
cd "/home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0" && CC="arm-openwrt-linux-muslgnueabi-gcc" CCSHARED="arm-openwrt-linux-muslgnueabi-gcc -DPIC -fpic" CXX="arm-openwrt-linux-muslgnueabi-g++" LD="arm-openwrt-linux-muslgnueabi-gcc" LDSHARED="arm-openwrt-linux-muslgnueabi-gcc -shared" CFLAGS="-Os -pipe -fno-caller-saves -fno-plt -fhonour-copts -mfloat-abi=hard -ffile-prefix-map=/home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0=fail2ban-1.1.0 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro" CPPFLAGS="-I/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/toolchain-arm_cortex-a7+neon-vfpv4_gcc-14.3.0_musl_eabi/usr/include -I/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/toolchain-arm_cortex-a7+neon-vfpv4_gcc-14.3.0_musl_eabi/include -I/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/toolchain-arm_cortex-a7+neon-vfpv4_gcc-14.3.0_musl_eabi/include/fortify -I/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/usr/include/python3.13" LDFLAGS="-L/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/toolchain-arm_cortex-a7+neon-vfpv4_gcc-14.3.0_musl_eabi/usr/lib -L/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/toolchain-arm_cortex-a7+neon-vfpv4_gcc-14.3.0_musl_eabi/lib -fuse-ld=bfd -znow -zrelro -lpython3.13" _PYTHON_HOST_PLATFORM="linux-arm" _PYTHON_SYSCONFIGDATA_NAME="_sysconfigdata__linux_arm-linux-musleabihf" PYTHONPATH="/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/usr/lib/python3.13:/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi//usr/lib/python3.13/site-packages:/home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0/ipkg-install//usr/lib/python3.13/site-packages" PYTHONDONTWRITEBYTECODE=1 _python_sysroot="/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi" _python_prefix="/usr" _python_exec_prefix="/usr"  CARGO_BUILD_TARGET=armv7-unknown-linux-musleabihf CARGO_HOME=/home/dwardo/GADGETS/ROUTEURS/openwrt/dl/cargo CARGO_PROFILE_RELEASE_CODEGEN_UNITS=1 CARGO_PROFILE_RELEASE_DEBUG=false CARGO_PROFILE_RELEASE_DEBUG_ASSERTIONS=false CARGO_PROFILE_RELEASE_LTO=true CARGO_PROFILE_RELEASE_OPT_LEVEL=z CARGO_PROFILE_RELEASE_OVERFLOW_CHECKS=true CARGO_PROFILE_RELEASE_PANIC=unwind CARGO_PROFILE_RELEASE_RPATH=false CARGO_TARGET_ARMV7_UNKNOWN_LINUX_MUSLEABIHF_LINKER=arm-openwrt-linux-muslgnueabi-gcc RUSTFLAGS="-Ctarget-feature=-crt-static -lssp_nonshared -Clink-arg=-fuse-ld=bfd" TARGET_CC=arm-openwrt-linux-muslgnueabi-gcc TARGET_CFLAGS="-Os -pipe -fno-caller-saves -fno-plt -fhonour-copts -mfloat-abi=hard -ffile-prefix-map=/home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0=fail2ban-1.1.0 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro " PYO3_CROSS_LIB_DIR="/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/usr/lib/python3.13" SETUPTOOLS_RUST_CARGO_PROFILE="release"    /home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/hostpkg/bin/python3.13  -m build --no-isolation --outdir "/home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0/"/openwrt-build --wheel   "/home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0/" 
* Getting build dependencies for wheel...

Traceback (most recent call last):
  File "/home/dwardo/GADGETS/ROUTEURS/openwrt/staging_dir/hostpkg/lib/python3.13/site-packages/pyproject_hooks/_impl.py", line 402, in _call_hook
    raise BackendUnavailable(
    ...<4 lines>...
    )
pyproject_hooks._impl.BackendUnavailable: Cannot import 'setuptools.build_meta'

ERROR Backend 'setuptools.build_meta:__legacy__' is not available.
make[2]: *** [Makefile:87: /home/dwardo/GADGETS/ROUTEURS/openwrt/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/fail2ban-1.1.0/.built] Error 1
make[2]: Leaving directory '/home/dwardo/GADGETS/ROUTEURS/openwrt/feeds/packages/net/fail2ban'
time: package/feeds/packages/fail2ban/compile#0.56#0.07#0.63
    ERROR: package/feeds/packages/fail2ban failed to build.
make[1]: *** [package/Makefile:187: package/feeds/packages/fail2ban/compile] Error 1
make[1]: Leaving directory '/home/dwardo/GADGETS/ROUTEURS/openwrt'
make: *** [/home/dwardo/GADGETS/ROUTEURS/openwrt/include/toplevel.mk:233: package/feeds/packages/fail2ban/compile] Error 2
```

### Confirmation Checklist

- [x] The package is maintained in this repository.
- [x] I understand that issues related to [the base OpenWrt repository](https://github.com/openwrt/openwrt/issues) or [LuCI repository](https://github.com/openwrt/luci/issues) will be closed.
- [x] I am reporting an issue for OpenWrt, not an unsupported fork.

--- a/net/fail2ban/Makefile
+++ b/net/fail2ban/Makefile
@@ -18,6 +18,10 @@ PKG_LICENSE:=GPL-2.0-or-later
 PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:fail2ban:fail2ban
 
+PKG_BUILD_DEPENDS:= \
+	python3/host \
+	python-setuptools/host
+
 include $(INCLUDE_DIR)/package.mk
 include ../../lang/python/python3-package.mk
 

